app/controllers/application_controller.rb

```diff
@@ -2,4 +2,20 @@
 
 # ApplicationController
 class ApplicationController < ActionController::Base
+  # before_action :update_allowed_parameters, if: :devise_controller?
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  private
+
+  def configure_permitted_parameters
```

Comment:
Can we permit these parameters in the devise registration controller?

Reply:
Done